script.js

```diff
@@ -0,0 +1,67 @@
+function clearBoard(){
+for (var i = 1; i < 10; i ++) {
+document.getElementsByClassName("grid-element").className = "white"
+}
+}
+
+var clearBoardListener = document.getElementById("clear");
+clearBoardListener.addEventListener("click", clearBoard)
+
+
+var winnerListener = document.getElementsByClassName("grid-element");
+document.getElementById("Box 1").className = "yellow"
+
+var
+
+
+var gameRecord = [null];
+function getWinner() {
+  if (gameRecord.end === "X") {
```

Comment:
I'm having trouble understanding what `gameRecord.end` does here. I think you would benefit from writing pseudo code and describing the problem's solution before writing code